NewForce-Cohort-4/nutshell-the-lucky-biscuits
# Cl deleted tasks

# Description
Enabled ability to view completed tasks. Layout adjustments. Percentage of tasks completed now shows
Fixes # (issue)
## Type of change
Please delete options that are not relevant.


# Testing Instructions
add tasks. mark task complete and make sure it is removed from task list. Go to completed tasks and uncheck. Make sure it is no longer shown in completed list an moves back to task list. Make sure percentage done updates accordingly
# Checklist:
